### ISSUE TITLE
[Bug]: Videos not playing and player window flickers.

### ISSUE THREAD

**Issue by ChronosNoob** [FreeTubeApp/FreeTube#4303]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

When attempting to play a video, the video will not load and may flicker.

NotSupportedError: Failed to load because no supported source was found. 

### Expected Behavior

Video is meant to load and not flicker.

### Issue Labels

content not loading, visual bug

### FreeTube Version

v0.19.1 Beta

### Operating System Version

Win 10 Home 19044.3086

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

N/A 

### Additional Information

<img width="405" alt="image" src="https://github.com/FreeTubeApp/FreeTube/assets/83444922/bf3b9453-4130-4516-8988-28e0b089121f">
The video will either flicker or infinitely load. The flickering is pretty headache inducing, it looks like the video is switching between a low resolution and high resolution. 
<img width="770" alt="image" src="https://github.com/FreeTubeApp/FreeTube/assets/83444922/37c72051-94f6-4c9e-9d56-212439b5b621">
I have not changed any settings within the app in the time between when I last used FreeTube and when this bug appeared. Also, as far as I am aware, I have not updated java or any other dependency that could be used. I made sure that I wasn't banned in any way from the youtube.com so I doubt this could be caused by some form of IP ban. 
<img width="900" alt="image" src="https://github.com/FreeTubeApp/FreeTube/assets/83444922/170bec66-5d78-4531-a769-70ea51868a05">
I have attempted changing format to no avail. All it does is remove the error shown at the bottom of my screen.
I have also attempted to use Invidious API as a proxy, this did not work either. It's unfortunate that I can't access videos but I am more concerned about the flickering. It causes me, a non photosensitive person, a headache, I am slightly worried that it might cause seizures in those who are prone to photosensitive seizures. Restarting the application does not work.

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by ChronosNoob** [comment]
Upon opening the application this error is also shown. 
<img width="419" alt="image" src="https://github.com/FreeTubeApp/FreeTube/assets/83444922/1690da0d-2cfc-4864-ab37-3d075d1d7326">

**Comment by ChronosNoob** [comment]
https://github.com/FreeTubeApp/FreeTube/assets/83444922/224f33f8-d242-44b8-8eb8-15535ed39737

The flickering continuing for a long period of time is unlikely but has happened a few times. Unfortunately, I have not been able to catch that happening with OBS on hand. However, I have attached a video of an instance where the flicker happens once. At it's worst it flickered about 15 times per second.

**Comment by Krysztophe** [comment]
I seem to have the same problem: massive flickering and the video does not play

Tested: v 0.19.1 and  0.19.1-nightly-3733

Config : Linux Debian 12 bookworm

The flickering seems to disappear or to appear only once if I choose by default the DASH format, 720p (360p was my default), but this not always the case.
Anyway, the video does not play.

I've teste all the settings that I could think of, without any success.

I have no idea how to access the dev tools to look at the error messages.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Does the flicker look like this?

https://github.com/FreeTubeApp/FreeTube/issues/1442#issuecomment-946151192

**Comment by ChronosNoob** [comment]
Yes

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of #3892 #1442
